subsys/bluetooth/shell/audio.c

```diff
@@ -795,13 +795,17 @@ static void add_sink(struct bt_audio_ep *ep, uint8_t index)
 {
 	shell_print(ctx_shell, "Sink #%u: ep %p", index, ep);
 
+	__ASSERT(index < CONFIG_BT_AUDIO_UNICAST_CLIENT_ASE_SNK_COUNT,
+		 "Too few sinks configured");
```

Comment:
This shouldn't be possible, as the number of sinks supported by the stack is `CONFIG_BT_AUDIO_UNICAST_CLIENT_ASE_SNK_COUNT`, and the index should never go above that. 

Is this something you have experienced? If so, that's a bug in the stack, not a problem in the shell application, and should be fixed in the stack instead.

Reply:
It's a warning/error when compiling with the newer SDK (0.16) where the compiler complains when the symbols are zero. I figured I'd generalize the assert instead of making it specific to zero.